mime.go

```diff
@@ -107,8 +120,96 @@ func (mh *mimeHandler) initJPEGEncoder(frame *C.AVFrame) error {
 	return nil
 }
 
+func (mh *mimeHandler) convertYUYV(frame *C.AVFrame) ([]byte, camera.ImageMetadata, error) {
```

Comment:
I notice no locks in this function; if it is meant only to be called from other functions handling a mutex lock, add a comment above it saying so.

Reply:
Please be sure to test after for hangs on module restart/closure.